helix-core/src/main/java/org/apache/helix/task/WorkflowContext.java

```diff
@@ -57,6 +57,9 @@ protected enum WorkflowContextProperties {
   // Otherwise, the context will not be written to ZK by the controller.
   private boolean isModified;
 
+  // Have fixed size history of Scheduled Workflow Tasks
+  private final static int SCHEDULED_WORKFLOW_HISTORY_SIZE = 100;
```

Comment:
Thanks for making the fix. For the size here, should we use something smaller like 20?

Reply:
Thanks for review. Reduced to 20.